docs/mixins/state.md

```diff
@@ -8,7 +8,7 @@ Use a `StateMixin` if your object/view needs to maintain information that isn't
 
 * [Using StateMixin](#using-statemixin)
 * [StateMixin's `StateModel`](#statemixins-statemodel)
-* [StateMixin's `stateDefaults`](#statemixins-statedefaults)
```

Comment:
Maybe move this item up about `StateModel`?  Both `StateModel` and `stateEvents` are options.  This item should probably just say `[Setting a default state]` vs `StateMixin's` as this isn't specific to the StateMixin